src/resources/css/rsvp.css

```diff
@@ -187,4 +187,18 @@
 .event-tickets-meta-label {
 	font-weight:700;
 	margin: 0 1em 0 0;
-}
\ No newline at end of file
+}
+
+/* TwentySixteen fix for box shadow as bottom border on <a> link */
+
+.entry-content a,
+.entry-summary a,
+.taxonomy-description a,
+.logged-in-as a,
+.comment-content a,
+.pingback .comment-body > a,
+.textwidget a,
+.entry-footer a:hover,
+.site-info a:hover {
+	box-shadow: none;
+}
```

Comment:
Doh! This is the generated CSS. But take it as a question about the .pcss source.